kafka/producer/base.py

```diff
@@ -60,11 +62,13 @@ def _send_upstream(queue, client, codec, batch_time, batch_size,
             # Adjust the timeout to match the remaining period
             count -= 1
             timeout = send_at - time.time()
-            msgset[topic_partition].append((msg, key))
+            msgset[topic_partition].append(msg)
```

Comment:
Why do we want to change this? Why not keep the dictionary key as a tuple of `(msg, key)`?